tests/foreman/cli/test_oscap.py

```diff
@@ -1127,6 +1128,48 @@ def test_positive_delete_scap_policy_with_name(self):
         with self.assertRaises(CLIReturnCodeError):
             Scapcontent.info({'name': scap_policy['name']})
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_associate_scap_policy_with_single_server(self):
+        """Assign an audit policy to a single server
+
+        :id: 30566c27-f466-4b4d-beaf-0a5bfda98b89
+
+        :setup:
+
+            1. Oscap should be enabled.
+            2. Oscap-cli hammer plugin installed.
+            3. At least 1 policy and host.
+
+        :steps:
+
+            1. Login to hammer shell.
+            2. Execute "policy" command with "update" as sub-command.
+            3. Pass host name as the parameter.
+
+        :expectedresults: The scap policy is updated.
+        """
+        host = entities.Host()
+        host.create()
+        name = gen_string('alpha')
+        scap_policy = make_scap_policy({
+            'name': name,
+            'scap-content-id': self.scap_id_rhel6,
+            'scap-content-profile-id': self.scap_profile_id_rhel6,
+            'period': OSCAP_PERIOD['weekly'].lower(),
+            'weekday': OSCAP_WEEKDAY['friday'].lower()
+        })
+        host_name = host.name + "." + host.domain.name
+        Scappolicy.update({
+            'id': scap_policy['id'],
+            'hosts': host_name,
+        })
+        data = entities.Policies(id=scap_policy['id']).read().hosts
+        self.assertNotEqual(len(data[0]['name']), 0,
```

Comment:
what is the expected value of data[0]['name'] ?

Reply:
Yes, makes sense. Would `self.assertNotEqual(data[0]['name'], '', ...)` work? Or are we able to get something even more specific?